src/main/java/chatbot/gui/DialogBox.java

```diff
@@ -0,0 +1,56 @@
+package chatbot.gui;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.Node;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextAlignment;
+
+import java.io.IOException;
```

Comment:
Similar issue as before, I believe that the import statement for 'java.io.IOException' should be in the 'STANDARD_JAVA_PACKAGE' group.